crates/sargon/src/wrapped_radix_engine_toolkit/low_level/manifest_summary/account_deposit/simple_fungible_resource_bounds.rs

```diff
@@ -0,0 +1,168 @@
+use crate::prelude::*;
+
+/// Represents the bounds for a simple fungible resource, which can
+/// be exact, at most, at least, between, or unknown amount.
+#[derive(Clone, Debug, PartialEq, Eq, uniffi::Enum)]
+pub enum SimpleFungibleResourceBounds {
+    Exact {
```

Comment:
any doc on each variant you can carry over from RET?

Reply:
No, only the enum docs. The variants are self-explanatory IMHO